### DISCUSSION TITLE
How to change key count in mania edit mode?

### DISCUSSION THREAD

**Discussion by KuribohG** [ppy/osu#16338] [Q&A] [answered]
It is 5 keys in default. How should I change the key count to 4?

**Comment by peppy** [comment, marked as answer]
I believe you can do this by changing the "object size" slider. You will need to exit and re-enter the editor after making the change.

We do need better support for this.

**Reply by peppy** [reply]
Also tracking this at https://github.com/ppy/osu/issues/16339.

**Reply by bdach** [reply]
was already tracked as #12904
